BipedRobot/ZMPBipedRobot/examples/1. Default Controller/simulation_controller.jl

```diff
@@ -0,0 +1,288 @@
+    dpi = dpi,
+)
+plt_τ = plot(
+    xlims = (0, tend),
+    #ylims = (-pi, pi),
+    dpi = dpi,
+    xlabel = L"$t$ [s]",
+    ylabel = L"$\tau$ [Nm]",
+    layout = (2, 2),
+)
+
+for (side_idx, side) in enumerate(["Left", "Right"])
+    for (joint, name) in enumerate(["Leg", "Knee"])
+        plot!(
+            plt_θ[joint, side_idx],
+            tplot,
+            qref[:, (2 * joint - 1) + (side_idx - 1)];
+            lw = lw,
+            label = "Reference",
+            title = "$(side) $(name)",
+        )
+        plot!(
+            plt_θ[joint, side_idx],
+            tsim,
+            qsim[:, dof_offset + (2 * joint - 1) + (side_idx - 1)];
+            lw = lw,
+            label = "Simulation",
+        )
+        str = latexstring("q_$(dof_offset + (2*joint - 1) + (side_idx - 1))") * " [rad]"
+        ylabel!(plt_θ[joint, side_idx], str)
+
+        str = latexstring("q̇_$(dof_offset + (2*joint - 1) + (side_idx - 1))") * " [rad/s]"
+        ylabel!(plt_ω[joint, side_idx], str)
+
+        str = latexstring("τ_$(dof_offset + (2*joint - 1) + (side_idx - 1))") * " [Nm]"
+        ylabel!(plt_τ[joint, side_idx], str)
+        plot!(
+            plt_τ[joint, side_idx],
+            tsim[1:len_t],
+            torque_sim[dof_offset + (2 * joint - 1) + (side_idx - 1), 1:len_sim];
+            lw = lw,
+            label = false,
+            title = "$(side) $(name)",
+        )
+        plot!(
+            plt_ω[joint, side_idx],
+            tsim,
+            vsim[:, dof_offset + (2 * joint - 1) + (side_idx - 1)];
+            lw = lw,
+            label = false,
+            title = "$(side) $(name)",
+        )
+        if (joint == 1)
+            plot!(plt_θ[joint, side_idx]; legend = false)
+        end
+    end
+end
+
+plt_com = plot(; xlabel = L"$t$ [s]", xlims = (0, tend), layout = (2, 1), dpi = dpi)
+
+plot!(
+    plt_com[1],
+    tsim[1:len_t],
+    CoMsim[1, 1:len_sim];
+    lw = lw,
+    label = "Simulated",
+    ylabel = L"$x$ [m]",
+    title = "CoMx",
+)
+plot!(plt_com[1], tplot, CoM[1, :]; label = "Predicted", lw = lw)
+plot!(
+    plt_com[2],
+    tsim[1:len_t],
+    CoMsim[3, 1:len_sim];
+    lw = lw,
+    label = "Simulated",
+    ylabel = L"$z$ [m]",
+    title = "CoMz",
+)
+plot!(plt_com[2], tplot, CoM[3, :]; label = "Predicted", lw = lw)
+
+ZMPref = reduce(hcat, zt.ZMP)
+right_plot = reduce(hcat, fp.right)
+left_plot = reduce(hcat, fp.left)
+plt_ZMP = plot(;
+    title = "ZMP in XY plane",
+    xlabel = L"$x$ [m]",
+    ylabel = L"$y$ [m]",
+    legend = true,
+    dpi = dpi,
+    #aspect_ratio =:equal,
+    #xlim = (-0.3, 1.5), ylim = (-0.3, 1.5)
+)
+scatter!(ZMPsim[1, :], ZMPsim[2, :]; ms = ms, msw = msw, label = "Measured ZMP")
+plot!(ZMPref[1, :], ZMPref[2, :]; label = "Reference", lw = lw)
+scatter!(left_plot[1, :], left_plot[2, :]; shape = :rect, label = "Left")
+scatter!(right_plot[1, :], right_plot[2, :]; shape = :rect, label = "Right")
+#plot!(legend=:outerbottom, legendcolumns=4)
+
+if PLOT_RESULT
```

Comment:
Here, instead of saving your plots in a folder, you could generate notebooks, or build a documentation

You could inspire yourself with https://github.com/dionysos-dev/Dionysos.jl/blob/master/docs/make.jl

This way, people would'nt have to download loads of images on their computer when cloning Dionysos